src/PipeManager.cs

```
@@ -35,14 +35,23 @@ public PipeManager(string name)
 
         public void WriteToPipe(string data)
         {
-            //THIS IS A GOD DAMN PROBLEM. I DON'T KNOW WHERE ITS HAPPENING BUT WHEN I ENCODE IN UTF8 IT GETS GARBLED
-            StreamWriter writer = new StreamWriter(pipe, Encoding.Default, 512, true);
-
-            writer.Write(data);
-            writer.Flush();
-            pipe.WaitForPipeDrain();
-
-            writer.Close();
+            int numberOfBytes = data.Length * sizeof(Char);
+            //THIS IS A GOSH DARN PROBLEM. I DON'T KNOW WHERE ITS HAPPENING BUT WHEN I ENCODE IN UTF8 IT GETS GARBLED
+            StreamWriter writer = new StreamWriter(pipe, Encoding.Default, numberOfBytes, true);
+            try
+            {
+                writer.Write(data);
+                writer.Flush();
+                pipe.WaitForPipeDrain();
+            }
+            catch (Exception e)
+            {
+                return;
+            }
+            finally
+            {
+                writer.Close();
+            }
```

Also, are you sure you simply want to swallow the `Exception e`? can we not dumpt it to debug or some kind of error stream?